sb_vision/tokens.py

```diff
@@ -179,65 +37,68 @@ def __init__(self, id: int, certainty: float=0.0) -> None:
         """
         self.id = id
         self.certainty = certainty
+        self.pixel_coords = pixel_coords
+        self.pixel_centre = pixel_centre
 
     @classmethod
     def from_apriltag_detection(
-        cls,
-        apriltag_detection: 'ApriltagDetection',
-        image_size: Tuple[int, int],
-        distance_model: Optional[str],
+            cls,
+            apriltag_detection: 'ApriltagDetection',
+            camera_model: Optional[str],
     ) -> 'Token':
         """Construct a Token from an April Tag detection."""
+
+        pixel_coords = np.array([list(l) for l in apriltag_detection.p])
+
+        # centre of marker: average the corners
+        pixel_centre = tuple(
+            np.average(np.array([[1, 2], [1, 2], [1, 2]]), axis=0))
+
+        marker_id = apriltag_detection.id
+
         # *************************************************************************
         # NOTE: IF YOU CHANGE THIS, THEN CHANGE ROBOT-API camera.py
```

Comment:
You have changed this file, however `robot-api` hasn't been changed (ideally by design). Is this note necessary anymore?

Reply:
I'm not sure that that's strictly true, and certainly nowhere near as dramatic as this comment appears.

_Adding_ a value here _makes it possible_ to also expose that value in robot-api. _Removing_ a value here is (hopefully obviously) a breaking change, so we'd need to modify the consumers of the library anyway.

I'd be in favour of reducing the shoutyness of this comment. However, from what I can see the comment itself doesn't change here, so that would be a separate PR anyway.